### PR TITLE
hping3: add page

### PR DESCRIPTION
**Pull request by ciph3rz**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/hping3.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+> Advanced ping utility which supports protocols such TCP, UDP, and Raw-IP.
+> Best run with elevated priviledges.
+> More information: <https://github.com/antirez/hping>.
+
+- Ping destination with 4 counts of ICMP requests:
+
+`hping3 --icmp --count {{4}} {{ip_or_hostname}}`
+
+- Scan for port 80 controlling which local source port 5090 initiates the scan:
```
[the remark this answers is not in the document]
Comment (ciph3rz) [reply pages/common/hping3.md:11]: specifying syn or ack scans through hping are specific to TCP handshake hence we don't need to call it out

File: pages/common/hping3.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+`hping3 --verbose --syn --destport {{80}} --baseport {{5090}} {{ip_or_hostname}}`
+
+- Traceroute to a specific port 80:
```
[the remark this answers is not in the document]
Comment (ciph3rz) [reply pages/common/hping3.md:15]: same as above

File: pages/common/hping3.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# hping3
+
+> Advanced ping utility which supports protocols such TCP, UDP, and Raw-IP.
+> Best run with elevated priviledges.
+> More information: <https://github.com/antirez/hping>.
+
+- Ping a destination with 4 counts of ICMP requests:
```
Comment (navarroaxel) [inline pages/common/hping3.md:7]:
```suggestion
- Ping a destination with 4 ICMP ping requests:
```

File: pages/common/hping3.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sbrl) [reply pages/common/hping3.md:11]: Yes, but in this example description you don't mention SYN or ACK.

File: pages/common/hping3.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sbrl) [reply pages/common/hping3.md:15]: Same as above

File: pages/common/hping3.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+> Advanced ping utility which supports protocols such TCP, UDP, and raw IP.
+> Best run with elevated priviledges.
+> More information: <https://github.com/antirez/hping>.
+
+- Ping a destination with 4 ICMP ping requests:
+
+`hping3 --icmp --count {{4}} {{ip_or_hostname}}`
+
+- Scan for port 80 controlling which local source port 5090 initiates the scan:
```
Comment (sbrl) [inline pages/common/hping3.md:11]:
```suggestion
- Scan TCP port 80, scanning from the specific local source port 5090:
```

File: pages/common/hping3.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+`hping3 --verbose --syn --destport {{80}} --baseport {{5090}} {{ip_or_hostname}}`
+
+- Traceroute to a specific port 80:
```
Comment (sbrl) [inline pages/common/hping3.md:15]:
```suggestion
- Traceroute using a TCP scan to a specific destination port:
```

File: pages/common/hping3.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+`hping3 --traceroute --verbose --syn --destport {{80}} --baseport {{5090}} {{ip_or_hostname}}`
```
Comment (sbrl) [inline pages/common/hping3.md:17]: I'm not sure specifying a specific source TCP port in all examples is a good idea.
```suggestion
`hping3 --traceroute --verbose --syn --destport {{80}} {{ip_or_hostname}}`
```

File: pages/common/hping3.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+- Perform an ACK scan to check if a given host is alive:
+
+`hping3 --count {{2}} --verbose --destport {{80}} -baseport {{5090}} -A {{ip_or_hostname}}`
```
Comment (sbrl) [inline pages/common/hping3.md:21]: Same as above.
```suggestion
- Perform a TCP ACK scan to check if a given host is alive:

`hping3 --count {{2}} --verbose --destport {{80}} -A {{ip_or_hostname}}`
```

**Review by sbrl — COMMENTED** [review]
Some of my comments from above were unaddressed. Also, I've left some additional comments below. Thanks for sticking with the review process!

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @ciph3rz!

**Pull request merged**